.github/workflows/sanity-productV2-skip.yml

```diff
@@ -0,0 +1,25 @@
+name: UI tests
```

Comment:
@Progi1984 to match the CI on other branches and to also match the protected branch rules I think we should rename this one to:

Reply:
```suggestion
name: UI tests Product V2
```

Reply:
And same in `sanity-productV2.yml` of course